docs/classic-ui/views.md

```diff
@@ -820,9 +820,36 @@ You need to get access to the view in your code if you are:
 -   calling a view from inside another view, or
 -   calling a view from your unit test code.
 
-Below are two different approaches for that.
+Below are three different approaches for that.
 
 
+(classic-ui-view-by-plone-api)=
+
+### By using `plone.api.content.get_view()`
+
+The `plone.api` provides a method to get a view by its registered name, the context, and the current request.
+
+```python
+from plone import api
+
+portal = api.portal.get()
+view = api.content.get_view(
+    name='plone',
+    context=portal['about'],
+    request=request,
+)
+```
+
+Since version 2.0.0, the request argument can be omitted.
```

Comment:
Let's add this admonition and its closing fence.

````suggestion
```{versionchanged} 2.0.0
Since version 2.0.0, the request argument can be omitted.
````